router/oauthResponseHandler/oauthResponseHandler.go

```diff
@@ -103,8 +103,8 @@ const (
 )
 
 type RefreshTokenBodyParams struct {
```

Comment:
This struct only exists for marshalling and sending the payload to control plane. Whether the type of `ExpiredSecret` is a `string` or a `[]byte` is not that much important, but changing seems to be appropriate since the underlying is a json

Reply:
The `ExpiredSecret` is actually a json column in control-plane DB. Initially we were stringifying this json in rudder-server, it didn't feel right as an extra processing would be required in control-plane to compare existing secret with the expired secret. Hence we are making this change to preserve the original data-type of `ExpiredSecret` and send it to control-plane

Let me know if it makes sense